app/views/campaign_log_entries/_form.erb

```diff
@@ -37,7 +37,7 @@
             <%= f.number_field :gp_gained, class: 'form-control', :step => 'any' %>
           </div>
           <div class="form-group col-md-2">
-            <%= f.label :downtime_gained, 'Downtime&nbsp+/-'.html_safe %>
+            <%= f.label :downtime_gained, 'Downtime' %>
```

Comment:
This was the only user-facing change in this PR. I didn't think the `+/-` was adding a ton of usability here and we were unable to assert it (tried "Downtime&nbsp+/-", "Downtime +/-", and "Downtime&nbsp;+/-"). Totally open to discussing this.

Reply:
Yeah, that seems fine. My original UX was based heavily on the log sheets themselves, so I carried over the +/- from there, but it's likely just clutter at this point.